pyoxidizer/src/distutils/unixccompiler.py

```diff
@@ -227,9 +227,15 @@ def extension_link_shared_object(self,
         if 'PYOXIDIZER_DISTUTILS_STATE_DIR' not in os.environ:
             raise Exception('PYOXIDIZER_DISTUTILS_STATE_DIR not defined')
 
+        libs=libraries.copy()
+        if 'crypto' in libs:
```

Comment:
the list of libraries that are statically built in should be obtained from `sys` or `sysconfig`

Reply:
Probably `sysconfig.get_config_vars('LOCALMODLIBS')` or MODLIBS